src/histolab/tiler.py

```diff
@@ -73,7 +73,9 @@ def locate_tiles(
         extraction_mask: BinaryMask = BiggestTissueBoxMask(),
         scale_factor: int = 32,
         alpha: int = 128,
-        outline: str = "red",
+        outline: Union[str, List] = "red",
+        linewidth: int = 1,
+        tiles: Iterable = None,
```

Comment:
I'd like to see what kind of object for the iterable e.g. `Iterable[str]` this way is more meaningful when we read the code and we know that we're expecting an iterable of `Tile` objects or something like that. Hope I have been clear 🤷🏽‍♂️

Reply:
OK, all addressed.